en/sensor/avanon_laser_interface.md

```diff
@@ -0,0 +1,56 @@
+# Avionics Anonymous Laser Altimeter Interface
+
+The [Avionics Anonymous Laser Altimeter Interface](https://www.tindie.com/products/avionicsanonymous/uavcan-laser-altimeter-interface/) allows a [number of common rangefinders](#supported_rangefinders) to be connected via the UAVCAN bus (this is a more robust interface than I2C).
+
+## Where to Buy
+
+* [AvAnon Laser Interface](https://www.tindie.com/products/avionicsanonymous/uavcan-laser-altimeter-interface/)
+
+## Supported Rangefinders {#supported_rangefinders}
+
+A full list supported rangefinders can be found on the link above. 
+
+At time of writing the following rangefinders are supported:
+
+- Lightware SF30/D
+- Lightware SF10/a
+- Lightware SF10/b
+- Lightware SF10/c
+- Lightware SF11/c
+- Lightware SF/LW20/b
+- Lightware SF/LW20/c
+
+
+## Pinouts
+
+### CAN Connector
+Pin | Name | Description
+--- | ---   | ---
+1   | POWER_IN | Power Supply. 4.0-5.5V supported, but must also be compatible with connected laser.
+2   | TX/SCL | TX for serial mode, Clock for I2C mode
+3   | RX/SDA | RX for serial mode, Data for I2C mode
+4   | GND | Signal/power ground.
+
+### Laser Connector
+
+Pin | Name | Description
+--- | ---   | ---
+1   | POWER_OUT | Filtered power at the supply voltage.
+2   | CAN+ | TX for serial mode, Clock for I2C mode
+3   | RX/SDA | RX for serial mode, Data for I2C mode
+4   | GND | Signal/power ground.
+
+
+## Wiring
+
+The rangefinder (laser) is connected to the AvAnon interface board, which is connected to one of the CAN ports on your autopilot.
+The wiring is as per the pinout above or using the included cables.
```

Comment:
What cables are included? Worth saying you can connect these rangefinders/boards "out of the box"

Reply:
Yes, good point. The unit can be purchased with the cables to connect to your laser and to the autopilot. I'll note that!